diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend allows for persistence of data and it allows us to validate everything coming from the client.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We do not use views because we are building an API, not a web app.
 ```
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.`